template/en/default/account/auth/login-small.html.tmpl

```diff
@@ -53,12 +53,15 @@
            name="Bugzilla_login"
            title="Login"
            placeholder="email address"
+           type="email"
```

Comment:
Yeah this seemed trivial enough that I figured it wasn't a big deal to sneak this in with the other changes.